pybamm/logger.py

```diff
@@ -1,11 +1,3 @@
-#
-# Logging class for PyBaMM
-# Includes additional logging levels inspired by verboselogs
-# https://pypi.org/project/verboselogs/#overview-of-logging-levels
-#
-# Implementation from stackoverflow
-# https://stackoverflow.com/questions/2183233/how-to-add-a-custom-loglevel-to-pythons-logging-facility
-#
```

Comment:
Should not remove this. Stackoverflow code should be attributed. This goes against our own guidelines in CONTRIBUTING.md haha!

Reply:
Contributing guidelines: "Snippets from blogs and [stackoverflow](https://stackoverflow.com/) can often be included without attribution"

Reply:
The coding guidelines say that we do not have to do the attribution